src/hooks/useToolbarGuards.ts

```diff
@@ -23,8 +23,7 @@ export function useConvertToVariable(range?: SourceRange) {
   }, [enable])
 
   useEffect(() => {
-    const parsed = parse(recast(ast))
-    if (trap(parsed)) return
+    const parsed = ast
```

Comment:
Are you sure it's safe to remove this? `parse(recast(ast))` updates the source ranges. Many of these calls were added to fix bugs caused by source ranges being outdated.

Reply:
It's hard for me to say if our Playwright suite would catch that sort of issue: wouldn't it require us to perform some action that relied on source ranges after a firing of `useToolbarGuards`?

Reply:
I do fear this will be the case, this parse(recast(ast)) was added [here](https://github.com/KittyCAD/modeling-app/pull/2033) with the constrain overlay work. I think if we see bad behavior occur it will be in the constrain buttons, the sketch constraint overlays, or in the selection highlighting, all after a codemod is performed.

Reply:
Upon manual smoke testing of the release, this does not seem to cause issues

Reply:
I tried applying a constraint in sketch mode and selecting/hovering afterward, as well as after applying an extrusion, and the source ranges seemed to align correctly.

Reply:
@jessfraz said it was ok